pypuppetdb/types.py

```diff
@@ -208,6 +221,25 @@ def __str__(self):
     def __unicode__(self):
         return self.__string
 
+    def metrics_dict(self):
```

Comment:
Altering the data here, means we are changing the spec that PuppetDB has set.  The application should decide what the data format should be.  If PuppetDB changes the wire format this will break.